src/libraries/Microsoft.Extensions.Hosting/tests/UnitTests/HostTests.cs

```diff
@@ -19,6 +19,18 @@ namespace Microsoft.Extensions.Hosting
 {
     public partial class HostTests
     {
+        [Fact]
+        public async Task StopAsyncWithCancellation()
+        {
+            var builder = new HostBuilder();
+            var host = builder.Build();
```

Comment:
```suggestion
            using var host = builder.Build();
```

> [EXECUTION TIMED OUT]
> Exit Code:-3Executor timed out after 900 seconds and was killed

I think since this test is actually starting the Host, the Host needs to be disposed so ConsoleLifetime doesn't [block the test exector's shutdown.](https://github.com/dotnet/runtime/blob/3aced828a9cd6efa5bb237db3e40aa8b93ecdfa4/src/libraries/Microsoft.Extensions.Hosting/src/Internal/ConsoleLifetime.cs#L86).

Reply:
good point!
my new test was without the "using/dispose" because  it was a copy/paste from one of the other tests....

a) i have added the "using" to the new test
b) i have added the using to the other tests inside the same .cs file
d) i have NOT modified the some of the other test classes where i have seen the same issue....

-------
Finding: the new test that i have added uses only the Hostbuilder-Constructor without any additional feature....
 I think (assumption, not verified) this host should NOT instantiate a ConsoleLifetime....

Some of the other tests is calling "Host.CreateDefaultBuilder" there maybe a ConsoleLifetime can be inside....